chapter-9-continuous-deployment/.vagrant/machines/node1/virtualbox/private_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
We should never upload a private key to a repo. Why did you add it?

Reply:
I have created a .gitignore file to exclude those files.